### ISSUE TITLE
[Bug] Tablet can be deposed into a Storage Scanner' container

### ISSUE THREAD

**Issue by ExDomino** [McJtyMods/RFTools#1740]
Issue description: 
The Storage Tablet with a Storage Control Screen Module can be moved when its GUI is open, deposed inside another wireless container, for example to recharge it. I have the bug when playing multiplayer.

Steps to reproduce:
1) Open a Storage Tablet linked to Storage Scanner;
2) You can deposit the tablet itself into a container linked to this Storage Scanner.

Versions:
* Minecraft: 1.12.2
* Forge: 2681
* CompatLayer (only if on Minecraft 1.10 or 1.11): N/A
* McJtyLib: 2.6.7
* RFTools: 7.33

Possibly incompatible mods and versions: OptiFine HD U D1 (not installed server-side)

Relevant logs, if any: There is no crash.

**Comment by McJty** [comment]
Indeed. But is this actually a problem? Nothing wrong seems to be happening here as far as I can see

**Comment by ExDomino** [comment]
This is a problem when you put your tablet into your tablet. It's against the rules of physics.
One of my player use that "bug" to recharge its tablet using a distant Energetic Infuser.

**Comment by McJty** [comment]
The tablet doesn't actually go into the tablet as it is not the tablet that contains the items.

**Comment by ExDomino** [comment]
No, but it's the tablet that open the GUI to access distants container.
It's the same thing. How can you put the tablet into a distant container? Because you open it to accces those distant containers. If you put it into them, the GUI should close, at least. But it's again against the rules of physics according to me.

**Comment by McJty** [comment]
I consider this a minor thing to be honest. Also who is talking about rules of physics in Minecraft anyway? Floating blocks? Infinite water sources?

**Comment by ExDomino** [comment]
Putting an object into itself, it's worst than floating blocks. But it's true that infinite water sources is a problem too, but there is nothing we can do about that.
